src/core/ComboBox/ComboBox.tsx

```diff
@@ -0,0 +1,370 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { Input, InputProps, Menu, MenuItem, SelectOption, Text } from '../..';
```

Comment:
I don't remember but importing like this `../..` might cause the issue for the client apps because of circular dependencies.

Reply:
So I need to import everything individually?

Reply:
ok changed